love-letter-app/src/app/ui/usablesvgs.tsx

```diff
@@ -0,0 +1,30 @@
+// here we will create an array with all svg names and then map over them to render them in the page
+'use client';
+import Image from "next/image";
+import { useState, useEffect, MouseEvent } from "react";
+
+export default function UsableSVGs( props: {setSelectedSVG: (svg: string) => void, selectedSVG: string} ) {
+    const svgNames = ["envelope.svg", "envelope1.svg"];
+
+
+    const handleSVGClick = (e: MouseEvent<HTMLImageElement>) => {
+        props.setSelectedSVG((e.target as HTMLImageElement).alt);
```

Comment:
use e.currentTarget since it's just an image with nothing else nested inside of it that could be triggerring the event.
and also remove the 'as HTMLImageElement' if you use e.currentTarget.